docs/source/markdown/podman-start.1.md

```diff
@@ -38,6 +38,10 @@ to run containers such as CRI-O, the last started container could be from either
 
 Proxy received signals to the process (non-TTY mode only). SIGCHLD, SIGSTOP, and SIGKILL are not proxied. The default is *true* when attaching, *false* otherwise.
 
+#### **-\-\all**, **-a**
```

Comment:
Please look really closely at this. Really, really, really closely. Compare it to the other lines, like line 37 or 32.

Reply:
:facepalm: 
THANKS